### ISSUE TITLE
Making the cursor blink

### ISSUE THREAD

**Issue by jatin-47** [hrushikeshrv/mjxgui#42]
It looks good when the cursor blinks.

**Comment by hrushikeshrv** [comment]
I agree. Currently the cursor is just a `|` character rendered by MathJax, and I'm not sure if we can/should be able to apply animations to it. I have noticed that the `|` character rendered by MathJax has a class of `mjx-c7C` applied, so maybe we could add an animation to that class through `mjxgui.css`, but I'm not sure if that will work and if it will be stable. But you can try it out and we will see how it turns out 👍🏻
